sources/packages/web/src/components/students/applicationTracker/InProgress.vue

```diff
@@ -0,0 +1,296 @@
+        edit your application for it to be processed again.</span
+      ></template
+    ></application-status-tracker-banner
+  >
+  <application-status-tracker-banner
+    v-if="
+      applicationDetails?.offeringStatus === OfferingStatus.ChangeOverwritten
+    "
+    label="Your institution updated the study period. Please re-submit your application with the updated study period."
+    icon="fa:fas fa-exclamation-circle"
+    icon-color="danger"
+    background-color="error_bg"
+    ><template #content
+      ><span
+        >Your institution updated the study period information. Please
+        <a href="" @click="goToStudentApplication" class="primary-color"
+          >start a new application</a
+        >, select the updated study period, and submit your application to have
+        it processed again. Please contact the Financial Aid Officer from your
+        institution if you require more information about the change in study
+        period.</span
+      ></template
+    ></application-status-tracker-banner
+  >
+  <application-status-tracker-banner
+    v-if="
+      applicationDetails?.exceptionStatus ===
+      ApplicationExceptionStatus.Declined
+    "
+    label="StudentAid BC has not accepted your application"
+    icon="fa:fas fa-exclamation-circle"
+    icon-color="danger"
+    content="StudentAid BC has not accepted 1 or more of the additional information that you provided in your 
+    application, making it ineligible for funding. Please contact StudentAid BC if your require more details. 
+    You may edit your application with new additional information to process it again or cancel your application."
+    background-color="error_bg"
+  />
+</template>
+<script lang="ts">
+import ApplicationStatusTrackerBanner from "@/components/students/applicationTracker/generic/ApplicationStatusTrackerBanner.vue";
+import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
+import { useRouter } from "vue-router";
+import {
+  ApplicationExceptionStatus,
+  InProgressApplicationDetailsAPIOutDTO,
+  OfferingStatus,
+  ProgramInfoStatus,
+} from "@/types";
+import { onMounted, ref, defineComponent } from "vue";
+import { ApplicationService } from "@/services/ApplicationService";
+
+export default defineComponent({
+  emits: ["declinedEvent"],
+  components: {
+    ApplicationStatusTrackerBanner,
+  },
+  props: {
+    applicationId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props, { emit }) {
+    const applicationDetails = ref<InProgressApplicationDetailsAPIOutDTO>();
+    const router = useRouter();
+
+    const goToStudentApplication = async () => {
+      router.push({
+        name: StudentRoutesConst.STUDENT_APPLICATION_FORM,
+      });
+    };
+
+    onMounted(async () => {
+      applicationDetails.value =
+        await ApplicationService.shared.getInProgressApplicationDetails(
+          props.applicationId,
+        );
+      // Any declined cards will call declined event.
```

Comment:
It does not seem to be something to be handled with an event. We probably need a sync-up in order to me better understand the reasons why it was necessary.